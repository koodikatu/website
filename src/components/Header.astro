<header class="w-full bg-white border-b-2">
  <nav class="w-full max-w-6xl mx-auto pr-4 flex justify-between items-center">
    <a href="/">
      <h1 class="sr-only">Koodikatu</h1>
      <img
        src="/koodikatu-square.png"
        alt="Koodikatu"
        class="w-20 aspect-square rounded"
      />
    </a>
    <ul class="flex items-center gap-4">
      <li class="hidden md:block"><a href="/">Home</a></li>
      <!-- <li><a href="/case-studies">Case Studies</a></li> -->
      <!-- <li><a href="/blog">Blog</a></li> -->
      <li>
        <a
          href="/contact"
          class="bg-gradient-to-tr from-purple-800 to-indigo-600 py-2 px-4 rounded text-slate-50 font-semibold"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</header>
