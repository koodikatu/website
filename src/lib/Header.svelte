<script lang="ts">
	import { page } from '$app/stores';
	import logo from '$assets/svelte-logo.svg';
</script>

<header />
