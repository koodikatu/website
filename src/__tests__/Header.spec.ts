import '@testing-library/jest-dom';
import { render } from '@testing-library/svelte';
import Header from '../lib/Header.svelte';

it('renders correctly', () => {
	const { container } = render(Header);
	expect(container).toMatchSnapshot();
});
