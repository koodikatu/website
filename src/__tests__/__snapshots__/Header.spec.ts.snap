// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<body>
  <div>
    <header
      class="svelte-16gzt7q"
    >
      <div
        class="corner svelte-16gzt7q"
      >
        <a
          class="svelte-16gzt7q"
          href="https://kit.svelte.dev"
        >
          <img
            alt="SvelteKit"
            class="svelte-16gzt7q"
          />
        </a>
      </div>
       
      <nav
        class="svelte-16gzt7q"
      >
        <svg
          aria-hidden="true"
          class="svelte-16gzt7q"
          viewBox="0 0 2 3"
        >
          <path
            class="svelte-16gzt7q"
            d="M0,0 L1,2 C1.5,3 1.5,3 2,3 L2,0 Z"
          />
        </svg>
         
        <ul
          class="svelte-16gzt7q"
        />
         
        <svg
          aria-hidden="true"
          class="svelte-16gzt7q"
          viewBox="0 0 2 3"
        >
          <path
            class="svelte-16gzt7q"
            d="M0,0 L0,3 C0.5,3 0.5,3 1,2 L2,0 Z"
          />
        </svg>
      </nav>
       
      <div
        class="corner svelte-16gzt7q"
      />
    </header>
  </div>
</body>
`;
